apps/bare-expo/ios/Podfile.lock

```diff
@@ -548,9 +548,9 @@ PODS:
   - React-jsinspector (0.71.0)
   - React-logger (0.71.0):
     - glog
-  - react-native-netinfo (9.3.5):
+  - react-native-netinfo (9.3.7):
```

Comment:
leftover of https://github.com/expo/expo/commit/e4d7e21f0c15d2488232bcd941e5619830f1b7e5